subsys/net/lib/config/init.c

```diff
@@ -249,6 +268,24 @@ static void ipv6_event_handler(struct net_mgmt_event_callback *cb,
 #if CONFIG_NET_CONFIG_LOG_LEVEL >= LOG_LEVEL_INF
 		NET_INFO("IPv6 address: %s",
 			 net_addr_ntop(AF_INET6, &laddr, hr_addr, NET_IPV6_ADDR_LEN));
+
+		if (ifaddr->addr_type == NET_ADDR_DHCP) {
+			char remaining_str[sizeof("01234567890")];
```

Comment:
Why the sizeof this string?

Reply:
Oh, it was just a copy paste of the code from [net shell](https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/ip/net_shell.c#L3468) where similar approach was taken, I didn't give too much though about it.

Indeed it might not be the best option, not very intuitive, I can rework this part to use "lifetime" string and it's size as a default, the buffer should be large enough to hold lifetime values (represented in seconds) in years.

Reply:
I've pushed an update.